### ISSUE TITLE
1.0.7 Helm Chart shows version as 1.0.6

### ISSUE THREAD

**Issue by RickLatrine** [istio/istio#13176]
**Describe the bug**
Helm upgrade Istio 1.0.6 to 1.0.7 does not work.
Helm Chart points to old version 1.0.6.

**Expected behavior**
Helm Chart.yml should point to correct versions 1.0.7.

**Steps to reproduce the bug**
Download versions from release page https://github.com/istio/istio/releases
or look into tagged commit at:
https://github.com/istio/istio/blob/1.0.7/install/kubernetes/helm/istio/Chart.yaml

**Version**
affected:  version 1.0.7 and 1.1.2

**Installation**

**Environment**

**Cluster state**

**Comment by howardjohn** [comment]
@RickLatrine thanks for pointing this out. This is just a cosmetic issue, right? The Helm chart says it is installing istio 1.0.6 but it is actually installing 1.0.7?

You can verify with `istioctl version --remote`. 

If it actually is not installing 1.0.7 then please let us know, that would be a P0 issue

**Comment by mgherman** [comment]
That syntax returns an error, using the packaged binary:
```~/Kubernetes/Nonprod/istio-1.0.7/bin$ ./istioctl version --remote
Error: unknown flag: --remote
```
helm ls indicates version 1.0.6 is still installed, and the bundled Chart.yml lists its version as 1.0.6

```NAME                             	REVISION	UPDATED                 	STATUS  	CHART                    	APP VERSION	NAMESPACE     
istio                            	3       	Wed Apr 10 03:23:48 2019	DEPLOYED	istio-1.0.6              	1.0.6      	istio-system  
```

https://github.com/istio/istio/blob/1.0.7/install/kubernetes/helm/istio/Chart.yaml#L3

However when reprovisioning a pod after the upgrade, I see 1.0.7 version proxies:

```Init Containers:
  istio-init:
    Container ID:  docker://a4c76fa009c48c1658823cc91199000ecd4a703664fd65b2feaa00d38fba2a3f
    Image:         docker.io/istio/proxy_init:1.0.7
    Image ID:      docker-pullable://istio/proxy_init@sha256:fa41549250eef900072bd96bbb9f45c8b35ae81465bfbc06c317e256d74a064d
```

```
  istio-proxy:
    Container ID:  docker://8e605a9d12e3df9db6f5e6d8cdca78d298d2c7d2a18d9f2e69b175d7899b7cac
    Image:         docker.io/istio/proxyv2:1.0.7
    Image ID:      docker-pullable://istio/proxyv2@sha256:9c6663cddbc984e88c27530d8acac7dca83070c4ad6d2570604cc4fff6c36a7a
    Port:          15090/TCP
    Host Port:     0/TCP
```

**Comment by RickLatrine** [comment]
As we are running multiple clusters, the Helm version is definitely not cosmetic.
I have to compare the Docker Image hashes to be safe now.

In order to increase the trust of deployment, I have patched the Chart.yml manually.
It seems to make sense to fork the repo and setup a template & build process for that.

Thanks for replies and to the Istio team to providing that security patch!

**Comment by howardjohn** [comment]
Thanks for the feedback, I didn't consider that.

I think the reason it didn't get updated is that we used the same git commit for 1.0.6 and 1.0.7, just updating the images.

@utka is it possible to change the release? I suspect its bad form/not possible from github to change a published release and I don't know if we would want a 1.0.8 just for this.

**Comment by RickLatrine** [comment]
This is also related to other branch/versions: e.g. 1.1 branch

**Comment by jwineinger** [comment]
I'd like to vote for a proper fix with a 1.0.8 release. We don't want to manually patch the chart just to use the new version with the security fixes. Not doing this makes a bit more work for us (and maybe other teams) to use. Incentivize people upgrading by making them as painless as possible.

**Comment by howardjohn** [comment]
sorry the 1.0 branch didn't get fixed properly, we will make sure this type of issue doesn't happen again moving forward
